internal/scheduler/submitcheck.go

```diff
@@ -214,20 +214,16 @@ func (srv *SubmitChecker) Check(ctx *armadacontext.Context, jobs []*jobdb.Job) (
 	return results, nil
 }
 
-func (srv *SubmitChecker) getIndividualSchedulingResult(jctx *context.JobSchedulingContext, state *schedulerState) schedulingResult {
+func (srv *SubmitChecker) getIndividualSchedulingResult(job *jobdb.Job, state *schedulerState) schedulingResult {
+	// Mark this job context as "not in a gang" for the individual scheduling check.
+	job = job.WithGangInfo(jobdb.BasicJobGangInfo())
```

Comment:
could make a `job.StripGangInfo()`.  Am 50:50 about whether this is actually better or not.